app/models/curator/filestreams/document.rb

```diff
@@ -9,11 +9,17 @@ class Filestreams::Document < Filestreams::FileSet
     belongs_to :file_set_of, inverse_of: :document_file_sets, class_name: 'Curator::DigitalObject'
 
     has_one_attached :document_primary
-    has_one_attached :document_access, service: :derivatives
+
+    with_options service: :derivatives do
+      has_one_attached :document_access
+      has_one_attached :text_plain
+    end
 
     has_paper_trail
 
     def required_derivatives_complete?(required_derivatives = DEFAULT_REQUIRED_DERIVATIVES)
+      return super(%i(characterization)) if text_plain.attached?
```

Comment:
While I don't _think_ there are any instances of this yet, we _might_ at some point have a situation where we have a PDF with an associated text file. So we might want to add more conditions here along the lines of:
```
return super(%i(characterization)) if text_plain.attached? && !(document_access.attached? || document_primary.attached?)
```